src/org/zaproxy/zap/extension/ascanrules/TestSQLInjection.java

```diff
@@ -1103,7 +1103,7 @@ public void scan(HttpMessage msg, String param, String origParamValue) {
 	
 						sqlInjectionFoundForUrl = true;
 	
-						continue; //to the next entry
+						break; //No further need to loop
```

Comment:
In this case it would stop anyway because of the `!sqlInjectionFoundForUrl` check. But, break is more clear (or less misleading).